components/setting_item_max.tsx

```diff
@@ -135,27 +77,38 @@ export default class SettingItemMax extends React.PureComponent {
         document.removeEventListener('keydown', this.onKeyDown);
     }
 
-    onKeyDown = (e) => {
-        if (this.props.shiftEnter && e.keyCode === Constants.KeyCodes.ENTER && e.shiftKey) {
+    onKeyDown = (e: KeyboardEvent) => {
+        const target = e.target as any;
```

Comment:
I couldn't find any other better solution

Reply:
Hey,
```
    onKeyDown = (e: React.KeyboardEvent<HTMLDivElement>) => {
        const target = e.target as HTMLElement;
```
The steps i did to figure it out.
- Checked online which class does tagName belong to since its one of the property being checked here. https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName
- Figured it belongs to Element
- Printed out the target on the console by running the app.
- Verified its a div element
- woila 🎉